src/utils/getPeopleFilteredAndSorted.ts

```diff
@@ -0,0 +1,63 @@
+import { Person, SortValues } from '../types';
+
+export function getPeopleFilteredAndSorted(
+  category: string,
+  query: string,
+  centuries: string[],
+  sort: string,
+  order: string,
+  people: Person[],
+) {
```

Comment:
Your function looks too complicated, split into two functions. The first filter and the second sort people